nautobot/extras/api/serializers.py

```diff
@@ -1087,7 +1088,7 @@ class Meta:
 
 
 class TagSerializer(NautobotModelSerializer):
-    tagged_items = serializers.IntegerField(read_only=True)
+    tagged_items = serializers.SerializerMethodField()
```

Comment:
Interested in why we have to do this here where in other places the previous pattern works okay.

Can you check if modifying any of the lines from [here](https://github.com/nautobot/nautobot/blob/7836cbe772a7283e945a8a34041dd1a6ef408900/nautobot/extras/api/views.py#L1221-L1224) will be sufficient?

Especially changing `queryset = Tag.objects.annotate(tagged_items=count_related(TaggedItem, "tag"))`
to
`queryset = Tag.objects.annotate(tagged_items=count_related(TaggedItem, "tag")).all()`